boards/avr-rss2/include/board.h

```diff
@@ -0,0 +1,116 @@
+/*
+ * Copyright (C) 2019 Robert Olsson <roolss@kth.se>
+ *
+ * This file is subject to the terms and conditions of the GNU Lesser
+ * General Public License v2.1. See the file LICENSE in the top level
+ * directory for more details.
+ */
+
+/**
+ * @ingroup     boards_avr-rss2
+ * @{
+ *
+ * @file
+ * @brief       Board definitions for the rss2 256rfr2 board.
+ *
+ * @author      Robert Olsson <roolss@kth.se>
+ *
+ */
+
+#ifndef BOARD_H
+#define BOARD_H
+
+#include "cpu.h"
+#include "periph/gpio.h"
+
+#ifdef __cplusplus
+extern "C" {
+#endif
+
+
+/**
+ * @name   LED pin definitions and handlers
+ * @{
+ */
+#define LED_PORT            PORTE
+#define LED_PORT_DDR        DDRE
+
+#define LED0_PIN            (1 << PE4) /* RED */
+#define LED1_PIN            (1 << PE3) /* YELLOW */
```

Comment:
Are those the same on my revision of the board?
when I do `saul write 1 1` in `examples/default` I'd expect the LED to light up.